File: Tema2/Python/mayormenor.py
def mayormenor(a,b,c):
    if(a>b and a>c):
        if(b>c):
            return f"Mayor: a: {a}  Menor: c: {c}"
        else:
            return f"Mayor: a: {a}  Menor: b: {b}"
    elif(b>a and b>c):
        if(a>c):
            return f"Mayor: b: {b}  Menor: c: {c}"
        else:
            return f"Mayor: b: {b}  Menor: a: {a}"
    else:
        if(a>b):
            return f"Mayor: c: {c}  Menor: b: {b}"
        else:
            return f"Mayor: c: {c}  Menor: a: {a}"


print("**********Mayor Menor**********")
print("Introduzca el valor de a: ")
a= int(input())
print("Introduzca el valor de b: ")
b= int(input())
print("Introduzca el valor de c: ")
c= int(input())
print(mayormenor(a,b,c))

File: Tema2/Typescript/mayormenor.ts
import readline from "node:readline/promises";
import { stdin as input, stdout as output} from "node:process";
const rl = readline.createInterface({input, output});

function mayormenor(a: number, b: number, c: number){
if (a > b && a > c) 
    if (b>c) 
        return `Mayor: ${a}  Menor: ${c}` 
    else 
        return `Mayor: ${a}  Menor: ${b}`;
    else if (b > a && b > c) 
        if (a>c) 
            return `Mayor: ${b}  Menor: ${c}` 
        else return `Mayor: ${b}  Menor: ${a}`;
    else if (a > b) 
        return `Mayor: ${c}  Menor: ${b}` 
    else return `Mayor: ${c}  Menor: ${a}`;
}

console.log("**********Mayor Menor**********")
const a = await rl.question("Introduzca el valor de a: ")
const b = await rl.question("Introduzca el valor de b: ")
const c = await rl.question("Introduzca el valor de c: ")

console.log(mayormenor(Number(a), Number(b), Number(c)))

rl.close();